radis/spectrum/operations.py

```diff
@@ -454,14 +454,16 @@ def multiply(s, coef, var=None, inplace=False):
     """
     # Check input
     var = _get_unique_var(s, var, inplace)
+    if Iunit == None:
+        Iunit = s.units[var]
 
     if not inplace:
         s = s.copy(quantity=var)
     #        if name is not None:
     #            s.name = name
 
     # Multiply inplace       ( @dev: we have copied already if needed )
-    w, I = s.get(var, wunit=s.get_waveunit(), copy=False)
+    w, I = s.get(var, wunit=s.get_waveunit(), copy=False, Iunit=Iunit)
     I *= coef  # @dev: updates the Spectrum directly because of copy=False
 
```

Comment:
and here, we should correct the unit of the spectrum.

- use `astropy.units` for the conversion. 

Hard part:
- how do you make sure that "mW/cm2/sr/nm" * "1" does not return something in ~ "W/m3/sr" ? Which is strictly speaking, through, but we should make these "cm2" are a distance and "nm" are wavelengths, and shouldn't be mixed. [spectral-flux-density](https://docs.astropy.org/en/stable/units/equivalencies.html#spectral-flux-luminosity-density-units) may help. I've never done that. 
- [ ] add tests! 

- Note: after #57  `coef` may have a dimension already.